_posts/2020-12-21-Bitcoin-Pump-adn-Dump.md

```diff
@@ -0,0 +1,29 @@
+---
+title:  Is Bitcoin a pump and dump scam?
+date: 2020-12-21 00:00:00 +0800
+author: allen
+---
+
+Whenever rich people tell me to invest in something and give improbable predictions of how much money something will be worth in one year, I tend to think “How is this a scam?” I remember in 2017 hearing Bitcoin will be worth 100K by the end of 2018 and saw a few friends of mine lose quite a bit of money after the crash from its peak of 19k. 
+<!--more-->
+
+<div style="text-align: center;">
+    <img src="/assets/blog/2020-12-21/bitconnect.gif" class="img-bordered">
+</div>
+
+We all remember the Bitconnect scammer yelling "BITCONNECT!!" before that cryptocurrency collapsed. I’m wondering if Bitcoin will collapse again with this spike and all this attention again
+
+# Early Days
+If you remember the early days of Bitcoin, and I’m not talking about 2017 when Bitcoin first became popular, you will remember hearing “Bitcoin is dead.” There were many ups and crashes in the price and with every crash everyone would forget about Bitcoin and move on.\
+The early days had many more crashes and people thinking this cryptocurrency thing was an interesting idea but wouldn’t be worth it in the long run. We have all heard the stories of people trading thousands of Bitcoins for pizza or something else. Many who worked in software didn’t sell, they bought more. 
+
+# Is it a scam?
+I think Bitcoin is the currency of the future but I’m afraid it is being used as a pump and dump scam. People aren’t using bitcoin as a currency at this moment but more of an investment.\
+When is the last time you used Bitcoin to buy something?\
+Most owners would probably say never. You would sell the Bitcoin and use the cash. This gives incentives for people to pump the price and sell. It isn’t a currency at this point, it is an asset just like a house or gold. Do you ever see anyone saying invest in the dollar? 
+
+# Is Bitcoin the future?
+I do still believe that Bitcoin will be the future unless regulations from governments get in the way. I do own Bitcoin and even at my job at this software development company in North Idaho, we have people who have invested early and refuse to sell.
+
+Until Bitcoin is used as a currency, it will be at risk of massive swings in price and manipulation. Bitcoin will try to stay at its natural price and I think it will continue to grow but there will still be the crashes.\
```

Comment:
```suggestion
Until Bitcoin is used as a currency, it will be at risk of massive swings in price and manipulation. Bitcoin will try to stay at its natural price and I think it will continue to grow but there will still be the crashes.

```
pull \ and add newline